debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -672,7 +672,17 @@ public void raiseError(Exception e) {
                         return;
                     }
                     try {
-                        SourceTaskContext taskContext = () -> offsetReader;
+                        SourceTaskContext taskContext = new SourceTaskContext() {
```

Comment:
So will this work with 1.x, too? We should set up CI jobs for testing against different versions, at least for testing 1.1.x in addition to 2.0.x at this point.

Reply:
Yes, it work with both versions. I suppose the daily running CI job would suffice for backward compatibility testing and Travis CI will run with 2.x.

Reply:
Ok; I take it you're going to set up a daily job for 1.1.1?